msteams-platform/sbs-tabs-and-messaging-extensions-with-sso.yml

```diff
@@ -147,7 +151,7 @@ items:
 
     1. Select **Channels** in the left pane. 
 
-    1. Select **Microsoft Teams** <img src="~/assets/images/bots/teamsicon.png" alt="Teams icon" width="20"/>.
+    1. Select **Microsoft Teams** :::image type="content"source="~/assets/images/bots/teamsicon.png" alt-text="Screenshot showing the home page.":::.
```

Comment:
For inline icons this is incorrect usage of Markdown syntax.

Reply:
The icon markdown rectified.